Makefile

```diff
@@ -64,6 +64,8 @@ clean: .env stop  ## | Stop and remove docker containers and artifacts.
 	${DC} rm -f
 	rm -fr .docker-build
 	rm -rf frontend/build/
+	mkdir -p frontend/build/static
+	touch frontend/build/static/.gitkeep
```

Comment:
What if instead of `touch` we did `git checkout frontend/build/static/.gitkeep` so it's "the same file"? Does that create problems with git workflow stuff?

Reply:
The question whether one empty file is "the same" as some other file is a bit philosophical, but I like this better anyway. I went with restoring the whole directory.

Reply:
Theseus' respository...